### PR TITLE
Release Notes for Dec 11, 2018

### PR DESCRIPTION
**Pull request by davidgardner11**
All CAM related: https://trello.com/b/Ry8kjOXN/release-notes-2018-12-11

### CODE REVIEW

File: Release Notes/2018/2018-12-11-cloud-platform-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+### Enhancements (7)
+
+#### [Cloud Application Manager](https://www.ctl.io/cloud-application-manager/)
+
+#### [Application Lifecycle Management](https://www.ctl.io/cloud-application-manager/application-lifecycle-management/)
+
+**Support for Google Compute Deployment Manager**
+
+[Application Lifecycle Management](https://www.ctl.io/cloud-application-manager/application-lifecycle-management/) now supports a new type of template box based on Google Deployment Manager Templates. Now you can create this new type of template box and add multiple Google Deployment templates files in YAML format to it to compose your box. Similar to the current support for AWS Cloud Formation, Azure ARM and Terraform templates, you can also define variables through Cloud Application Manager user interface, and then reference them from any Google Deployment Manager template file in the instance, with the usual Jinja format. For more information, please refer to [https://www.ctl.io/knowledge-base/cloud-application-manager/automating-deployments/google-deployment-manager/](https://www.ctl.io/knowledge-base/cloud-application-manager/automating-deployments/google-deployment-manager/)
```
Comment (jim-phillips) [inline Release Notes/2018/2018-12-11-cloud-platform-release-notes.md]: Links to other KB articles should use the relative path to the .md file

File: Release Notes/2018/2018-12-11-cloud-platform-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+**Improve new instance detection procedures**
+
+[Application Lifecycle Management](https://www.ctl.io/cloud-application-manager/application-lifecycle-management/) has improved its new instance detection procedure, to avoid automatic registration of cloned machines under the former existing instance. The auto-detection and registering will still work as usual for auto-scaling out events, but will prevent any other unwanted use case to be self-registered.
+
+**New system variable service.provider_type**
+
+[Application Lifecycle Management](https://www.ctl.io/cloud-application-manager/application-lifecycle-management/) now supports a new Jinja System Variable to allow access the provider type where the instance has been deployed to. You can refer to this new variable through the usual Jinja syntax by referencing: `{{ service.provider_type }}`. The returned value will be the corresponding descriptive name of the provider type, for example: "Amazon Web Services".
```
Comment (jim-phillips) [inline Release Notes/2018/2018-12-11-cloud-platform-release-notes.md]: `{{` must be `\{{`

**Comment by jim-phillips** [comment]
@davidgardner11 Please update per my comments

File: Release Notes/2018/2018-12-11-cloud-platform-release-notes.md (hunk 1 above)
Comment (davidgardner11) [reply Release Notes/2018/2018-12-11-cloud-platform-release-notes.md]: Hi Jim, just saw this - I'll address these issues. and resubmit the PR

Comment (davidgardner11) [reply Release Notes/2018/2018-12-11-cloud-platform-release-notes.md]: ok. fixed.

File: Release Notes/2018/2018-12-11-cloud-platform-release-notes.md (hunk 2 above)
Comment (davidgardner11) [reply Release Notes/2018/2018-12-11-cloud-platform-release-notes.md]: ok. fixed

**Pull request merged**